src/styles/utils/getUserListItemStyles/index.native.ts

```diff
@@ -0,0 +1,5 @@
+import type GetContextMenuItemStyle from './types';
+
+const getUserListItemStyles: GetContextMenuItemStyle = (styles) => [styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStretch, styles.optionRow];
```

Comment:
Can we create a `baseStyles.ts` file with all the common styles to make it clear what's different between web and mobile. I think in general we have developed too big an appetite for platform differences in our codebase.